src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -126,57 +125,36 @@ def run(self, revision):
             # Update to the target revision
             self.hg.update(rev=revision.mercurial, clean=True)
 
-            # Get the parents revisions
-            parent_rev = 'parents({})'.format(revision.mercurial)
-            parents = self.hg.identify(id=True, rev=parent_rev).decode('utf-8').strip()
-
-            # Find modified files by this revision
-            modified_files = []
-            for parent in parents.split('\n'):
-                changeset = '{}:{}'.format(parent, revision.mercurial)
-                status = self.hg.status(change=[changeset, ])
-                modified_files += [f.decode('utf-8') for _, f in status]
-            logger.info('Modified files', files=modified_files)
-
-        # List all modified lines from current revision changes
-        patch = Patch.parse_patch(
-            self.hg.diff(change=revision.mercurial, git=True).decode('utf-8'),
-            skip_comments=False,
-        )
-        modified_lines = {
-            # Use all changes in new files
-            filename: diff.get('touched', []) + diff.get('added', [])
-            for filename, diff in patch.items()
-        }
-
-        with stats.api.timer('runtime.mach'):
-            # mach configure with mozconfig
-            logger.info('Mach configure...')
-            run_check(['gecko-env', './mach', 'configure'], cwd=self.repo_dir)
-
-            # Build CompileDB backend
-            logger.info('Mach build backend...')
-            cmd = ['gecko-env', './mach', 'build-backend', '--backend=CompileDB']
-            run_check(cmd, cwd=self.repo_dir)
-
-            # Build exports
-            logger.info('Mach build exports...')
```

Comment:
Great, so this is going to fix #824 too!

Reply:
Yes 😄